prow/cluster/BUILD

```diff
@@ -1,18 +1,185 @@
 load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+load("@io_bazel_rules_k8s//k8s:objects.bzl", "k8s_objects")
 
 # Usage:
 #   bazel run //prow/cluster:foo.apply
 
 # TODO(fejta): consider k8s_deploy alias
 # https://github.com/bazelbuild/rules_k8s#aliasing-eg-k8s_deploy
 
+k8s_objects(
+    name = "production",
+    objects = [
+        ":branchprotector_cronjob",
+        ":deck_deployment",
+        ":deck_service",
+        ":hook_deployment",
+        ":hook_service",
+        ":horologium_deployment",
+        ":ingress",
+        ":jenkins_deployment",
+        ":jenkins_service",
+        ":lego_deployment",
+        ":mem_limit_range",
+        ":plank_deployment",
+        ":prow_job",
+        ":push_gateway",
+        ":sinker_deployment",
+        ":splice_deployment",
+        ":tide_deployment",
+        ":tide_service",
+        ":tot_deployment",
+        ":tot_service",
```

Comment:
You can nest `k8s_objects` in other `k8s_objects`, which enables you to break this sort of long list into more manageable pieces.

Reply:
Done